address-finder/address-finder-service.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const service = require('../questionnaire/request-service')();
```

Comment:
Thoughts on changing this to:
`const {get} = require('../questionnaire/request-service')();`